src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -97,28 +112,69 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('meta_title', TranslatableType::class, [
                 'type' => TextWithLengthCounterType::class,
+                'required' => false,
                 'options' => [
-                    'max_length' => 70,
+                    'max_length' => 255,
```

Comment:
i don't like hardcoding this constraint into form, any suggestion for the right place to extract it?

Reply:
I think i't's fine 😄 because this is a Form. It's display configuration, and it's in the right place.
You can put it in a constant at the top of the class but I'm not sure it will bring a lot of value.

Extracting a number is useful if you reuse this number in your app. But 255 is an arbitratry setting here. I dont think it will be targeted elsewhere.

Reply:
i've copied it from `Category` object model `$definition`, so it's used for validation, we could extract that.